Sources/Amplitude/Plugins/iOS/iOSUserInteractionCapture/UserInteractionCaptureDelegate.swift

```diff
@@ -0,0 +1,165 @@
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+
+import UIKit
+
+// MARK: File-private Variables
+
+private var globalUIPressGestureRecognizer: GlobalUIPressGestureRecognizer?
+private var globalUISlideGestureRecognizer: GlobalUISlideGestureRecognizer?
+private var globalUITextFieldGestureRecognizer: GlobalUITextFieldGestureRecognizer?
+
+// MARK: -
+
+internal final class UserInteractionCaptureDelegate {
+
+    // MARK: - Properties
+
+    weak var amplitude: Amplitude?
+
+    /// The ket window of the application represented as a `UIView` element.
+    var keyWindow: UIView? {
+        guard
+            let windowScene = applicationHandler()?.connectedScenes.first as? UIWindowScene,
+            let keyWindow = windowScene.windows.first(where: { $0.isKeyWindow })
+        else {
+            return nil
+        }
+        return keyWindow
+    }
+
+    /// The accessibility targets within `keyWindow`.
+    var accessibilityTargets: [AccessibilityTarget] {
+        guard let keyWindow else { return [] }
+        return accessibilityHierarchyParser.parseAccessibilityElements(in: keyWindow)
```

Comment:
Good call! Looking at it again, I think we can just get the first element under the tap - my initial concern was that multiple elements might be tapped with a single tap, but that doesn't seem to be an issue anymore since I'm already getting the first one.